### ISSUE TITLE
Make CoyIM look better

### ISSUE THREAD

**Issue by olabiniV2** [coyim/coyim#554]
Several people have thoughts and ideas on how Coy could look better. We now have a better understanding of how to use CSS to work with GTK applications, so we could potentially do this. @sacurio and @peterpalau have thoughts on this.

**Comment by peterpalau** [comment]
We could start with small changes based on a prototype and make some tests with technical and non-technical users to improve it during development. Yeah, I have some pretty ideas for this.

**Comment by sara-la-loca-de-los-gatos** [comment]
For me, the GUI in general looks kinda old fashioned and it gives me the feeling of those internal chats companies usually have that are not necessarily nice or friendly in any way. They just allow you to chat. Period.

IN MY OPINION, it doesn't look attractive, maybe adding some color to the design would make a big difference!

**Comment by sacurio** [comment]
I totally agree. We can capitalize the knowledge used in Wahay in order to improve the GUI in CoyIM. That was the original idea :)
